server/lib/routes/match.js

```diff
@@ -1291,7 +1295,70 @@ router.post('/matches', (req, res) => {
       }
     }
   }
-});
+
+  function transformToFhirObject(matchResults) {
+    const bundle = {};
+    bundle.resourceType = 'Bundle';
+    bundle.id = uuid4();
+    bundle.meta = {};
+    bundle.meta.lastUpdated = new Date().toISOString();
+    bundle.type = 'searchset';
+    bundle.total = 0;
+    bundle.entry = []; 
+  
+    const serverUrl = 'http://clientregistry.org/openmrs';
+    const family = {family: ''};
+    const given = {given: ''};
+    const system = {system: 'phone'};
+    const url = {url: 'http://hl7.org/fhir/StructureDefinition/match-grade'};
+    
+    function createPatientEntry(patient, matchGrade, score) {
```

Comment:
@mherman22 instead of adding individual fields that we have. Could you query the fhir server using the uuid per patient and just add the returned patient (just modify and add the score to this)